### DISCUSSION TITLE
Strict search on a "sentence" w/ FindMultiple — missing hoped-for result

### DISCUSSION THREAD

**Discussion by victorDECLAU** [stephanrauh/ngx-extended-pdf-viewer#1602] [Q&A] [answered]
Hello 👋 , 

I'm currently struggling with `ngxExtendedPdfViewerService.findMultiple(['cats and dogs'], options)`
My intention is to make the search of the string `'firstWord secondWord'` strict, as it is when using of WholeWord option on a regular `.find()` call. 

However, the space in the string seems to act like a separator, so the results of my search are equivalent to the results produced by a `.findMultiple(['cats',  'and, 'dogs'], options);

My architecture doesn't allow me to use the regular find method for sentences like 'cats and dogs', as I need to keep ongoing multi-search results available to the user. Typical findMultiple use in my case would be `.findMultiple(['birds', 'narval', 'cats and dogs'], options)` and I only want 'cats and dogs' strict sentence occurrences to be found.

What I tried so far : 
- Replacing spaces with ASCII `\xA0` code -> not working
- Setting `WholeWorld` option to `true` -> not working

Is there a fine tuning, or pure code workaround i'm missing to make this happen ? 

Versions used : 
- angular 12.2.18
- ngx-extended-pdf-viewer 15.0.9

Thanks for reading, 
Victor

**Comment by stephanrauh** [comment, marked as answer]
You've found a bug! When you open the find bar of the UI, you'll see the feature is broken there, too.

BTW, the "findMultipleSearchTexts" feature separates search terms by `\n`, not `\r` (which is `0x0A`, not `0xA0`). You were close! However, the idea of the API is to hide this arcane technical detail from you.

**Reply by stephanrauh** [reply]
I've opened a bug ticket: #1607

**Reply by victorDECLAU** [reply]
Thank you very much for the quick answer Stephan. 

You mention a difficult merge conflict resulting in this behavior : would you happen to know in between which versions the regression occurred ? 
I'm considering downgrading temporarily to bypass the issue.

Best regards

**Reply by victorDECLAU** [reply]
Just found out v13.5.3 delivers hoped-for results :)

**Reply by stephanrauh** [reply]
Yeah, I guess that bit of insight helps me. Thanks!

**Reply by stephanrauh** [reply]
Your bugfix has landed with version 15.1.1.

**Reply by victorDECLAU** [reply]
Thank you very much for you quick fix. It works perfectly :)
